src/psse_export.jl

```diff
@@ -59,6 +61,28 @@ function with_units(f::Function, sys::System, units::Union{PSY.UnitSystem, Strin
     end
 end
 
+# TODO a bit hacky, there may be a better way (round trip JSON serialization is too slow).
```

Comment:
@daniel-thom thoughts on this?

Reply:
If people need this use case, I'm fine with it. Copying large numbers of time series arrays can be slow. Does it work, though? I'm wondering if the copied components have a reference to the original TimeSeriesManager. This code runs when you add a component to the system.
```
    refs = SharedSystemReferences(;
        time_series_manager = data.time_series_manager,
        supplemental_attribute_manager = data.supplemental_attribute_manager,
    )
    set_shared_system_references!(component, refs)
```
The data will not be copied. But the component would have a reference to another system's time series manager.

Reply:
Some options:
1. Remove the shared references between system and components. It causes subtle bugs like this. Benefits are not worth the risks. The Python version does not have them.
2. Reorganize the shared references object. `SystemData` could have a field called `shared_system_references` which contains `time_series_manager` and `supplemental_attribute_manager`. When each component is added, it gets assigned a reference to that field. Then the time_series_manager is hot-swappable.
3. Keep the existing code but add a loop over all components and reassign `time_series_manager` and `supplemental_attribute_manager`.

Reply:
Strictly speaking, for my use case it should not be a problem that the resulting system's components have references to another system's time series manager, because systems I copy this way ought never be exposed to the user and my private use of them does not involve touching that stuff. But it's a sufficiently weird result that I'm willing to change it anyway. I don't really have the time/appetite to go mucking about with how we handle shared references right now (and it seems like an implementation detail we could easily change later anyway), so I'll go with option 2.

Reply:
Done